abci/example/kvstore/kvstore.go

```diff
@@ -68,6 +69,23 @@ func (app *KVStoreApplication) Info(req types.RequestInfo) (resInfo types.Respon
 	return types.ResponseInfo{Data: fmt.Sprintf("{\"size\":%v}", app.state.Size)}
 }
 
+// handle evidence
+func (app *KVStoreApplication) BeginBlock(req types.RequestBeginBlock) types.ResponseBeginBlock {
```

Comment:
So the PersistentKVStore does validator updates. Maybe this would be better handled there and keep the normal kvstore super simple?